WebContent/WEB-INF/jsp/_watchList.jsp

```diff
@@ -288,6 +288,7 @@
           
           if (owner) {
               show("p"+ pointId +"MoveUp");
+              show("p"+ pointId +"ShowDetails");
```

Comment:
I would prefer there were no changes in the old UI because I cannot predict the consequences of such a change and it will be very difficult to improve the operation with the use of the dojo framework, different versions of jquery, and jquery plug-ins.